lib/common.sh

```diff
@@ -279,8 +279,8 @@ elif [[ "${CAPM3RELEASEBRANCH}" = "release-1.6" ]]; then
 elif [[ "${CAPM3RELEASEBRANCH}" = "release-1.7" ]]; then
   export CAPM3_IMAGE=${CAPM3_IMAGE:-"${CONTAINER_REGISTRY}/metal3-io/cluster-api-provider-metal3:release-1.7"}
   export IPAM_IMAGE=${IPAM_IMAGE:-"${CONTAINER_REGISTRY}/metal3-io/ip-address-manager:release-1.7"}
-  export BARE_METAL_OPERATOR_TAG="v0.6.0"
-  export KEEPALIVED_TAG="v0.6.0"
+  export BARE_METAL_OPERATOR_TAG="v0.6.1"
+  export KEEPALIVED_TAG="v0.6.1"
   export IRONIC_TAG="v24.1.1"
   export BMOBRANCH="${BMORELEASEBRANCH:-release-0.6}"
```

Comment:
I believe we need to change this to use the tag instead as well. /cc @Rozzii

Reply:
Sorry, could you please elaborate what needs to change? I don't yet understand :sweat_smile: Or was that a message to @Rozzii ?

Reply:
The line where I commented i.e. `BMOBRANCH`, it should not point to release branch when the image is pointing to a specific tag, `BMOBRANCH` should also point to the tag